src/core/public/overlays/sidecar/components/sidecar.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useCallback, useMemo } from 'react';
+import useObservable from 'react-use/lib/useObservable';
```

Comment:
`yarn osd bootstrap` will fail because of this change. All import way of useObservable OSD is `import useObservable from 'react-use/lib/useObservable';`

Reply:
I see, thanks for the clarification, I didn't know that before 👍 Did a few investigation, it seems the types of `react-use` didn't work well with typescript.